### ISSUE TITLE
PHP native sessions 0.65 broke our workflow

### ISSUE THREAD

**Issue by LSIT-UCB** [pantheon-systems/wp-native-php-sessions#87]
Pantheon database workflows began failing with the latest update and clones of database and files failed.

session_start(): Failed to initialize storage module: user (path: ) wp-content/plugins/cas-maestro/cas-maestro.php:107

Cannot redeclare class Pantheon_Sessions wp-content/plugins/wp-native-php-sessions/pantheon-sessions.php:17

Once backed out to 0.64, workflow errors disappeared.

Thanks,
Michael

**Comment by danielbachhuber** [comment]
Hey @LSIT-UCB,

Could you open a ticket with CSE? It seems like this issue might be specific to your site configuration.

**Comment by hengkit** [comment]
@LSIT-UCB If you provide me with your email address or a site ID, I can open a ticket proactively for you.

**Comment by LSIT-UCB** [comment]
CSE?  I'm sorry, I am not familiar with that abbreviation.

Pantheon live support (Dustin) suggested I report the issue here.

-Michael

**Comment by LSIT-UCB** [comment]
Something about "the session might still be getting declared outside the bootstrap 😦 That means it might just be incompatible with the platform"

And that "The changelog for 0.6.5 makes sense here:

    Disables session instantiation when defined( 'WP_CLI' ) && WP_CLI because sessions don't work on CLI."

**Comment by kbest-mns** [comment]
@danielbachhuber I am seeing the same workflow error on a couple of our sites using this plugin since the 0.6.5 update. Similar to @LSIT-UCB I first opened a ticket with CSE and was then asked to report here.

After updating, the error:

`session_start(): Failed to initialize storage module: user (path: )`

...was caused by this line in a custom plugin we use:

`if ( !session_id() ) session_start();`

Our custom plugin has not changed in some time, and was working without any issues prior to the update. Reverting Native Sessions to 0.6.4 resolves the issue and allows us to deploy without workflow errors.

Ticket/case # is 99040 if that helps. I can provide additional info if needed.
